gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -740,6 +741,32 @@ def call_dd(self, _):
                 "No coin selected. Use 'load' to load the coin you want to look at.\n"
             )
 
+    def call_pred(self, _):
+        """Process pred command"""
+        if self.source != "cg":
```

Comment:
I would add there additional case. When user haven't load any coin in previous step it should return information, that he should do that to access this pred menu.

Reply:
Yup.  Will do.